docs/reference/contributing/guidelines/workflow.md

```diff
@@ -98,6 +98,12 @@ Any change in the functionality, it can be adding a new feature, adding a new me
 
 A feature contribution contains a new API, capability or behavior. It does not break backward compatibility with existing APIs, capabilities or behaviors. New feature contributions are very welcome in Mbed OS. However, because they add capability to the codebase, it's easy for a new feature to introduce bugs and a support burden. The introduction of new features should also come with documentation, majority of targets support and comprehensive test coverage proving the correctness of the feature per the documentation. Feature PRs are treated cautiously, and new features require a new minor version for the codebase. Features are candidates for feature releases. 
 
+Every functionality change pull request must contain release notes section addition to describe the change to users.
```

Comment:
must contain release notes section -> must contain a release notes section
I would also propose a title for this section 
so it would be:
must contain a release notes section called "Important changes" 

"important changes" is a placeholder actual name still needs to be defined

Reply:
fixed, adding "Release notes" section for now